tortoise_interpreter.py

```diff
@@ -4,7 +4,7 @@
 DRAW = "Draw a line"
 
 
-def main_tortoise(program, print_to_screen=print):
+def main_tortoise(program, args, print_to_screen=print):
```

Comment:
So this is fine, but I think there's a better way. At the minute, you're passing `args`, which contains all the arguments. At the moment, `main_tortoise` needs all those arguments. Maybe it won't always; maybe in the future there will be arguments that are only used by `tortoise.py`. Then you won't want to pass those in.

Also, `args` could contain anything. It's difficult to scan the function signature and understand _exactly_ what you need to pass in to make it work.

It would be better to make it explicit:
`def main_tortoise(program, filename, verbose, print_to_screen=print):`